src/main/java/step3/models/Game.java

```diff
@@ -0,0 +1,48 @@
+package step3.models;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import step3.utils.RandomSingleton;
+
+public class Game {
+    static final String CREATE_ERROR_MSG = "횟수에 음수는 들어올 수 없습니다.";
+    private final int carNumber;
+    private final int tryNumber;
+
+    public Game(int carNumber, int tryNumber) {
+        if (carNumber < 0 || tryNumber < 0) {
+            throw new IllegalArgumentException(CREATE_ERROR_MSG);
+        }
+
+        this.carNumber = carNumber;
+        this.tryNumber = tryNumber;
+    }
+
+    public List<List<Car>> play() {
+        List<List<Car>> results = new ArrayList<>();
+
+        results.add(initCars());
+
+        for (int i = 0; i < tryNumber; ++i) {
+            List<Car> previousCars = results.get(i);
+            results.add(moveCars(previousCars));
+        }
+
+        return results.stream().skip(1).collect(Collectors.toList());
```

Comment:
```suggestion
        results.add(initCars());

        for (int i = 0; i < tryNumber; ++i) {
            List<Car> previousCars = results.get(i);
            results.add(moveCars(previousCars));
        }

        return results.stream()
            .skip(1)
            .collect(Collectors.toList());
```
개인적으로 stream을 사용할 때는 개행을 해주는 편이 읽기 편한 것 같아요. 우혁님의 생각은 어떠신가요?

Reply:
아하 저는 한줄로 할만큼 짧은 수준이라면 한줄에 쓰고, 내용이 많을때만 개행을 하긴 하는데요. 사실 항상 개행하는게 읽기 편하긴 할 것 같습니다 :)